canonical-rails.gemspec

```diff
@@ -16,10 +16,13 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib}/**/*"] + ["MIT-LICENSE", "Rakefile", "README.md"]
 
-  s.add_dependency 'rails', '>= 4.1', '<= 7.1'
+  %w[actionmailer activerecord railties].each do |dep|
+    s.add_dependency dep, '>= 4.1', '<= 7.1'
+  end
+
+  s.add_dependency "sprockets-rails", '~> 3.0'
```

Comment:
Why is `sprockets-rails` a dependency?

Reply:
Because it was lol. I just halted the dependencies this pulled in to Rails 4.2 and nowhere beyond that. It can be thinned further!

Reply:
https://github.com/jumph4x/canonical-rails/pull/80/files#diff-576f477ee70cb313e97ed5a598455f0eed64184aaa963f8d759ff3c58507f4c3L23